File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.java
class Solution {
    public String removeStars(String s) {
        Stack<Character> stack=new Stack<>();
        for(int i=0;i<s.length();i++){
            if(stack.size()>0 && s.charAt(i)=='*'){
                stack.pop();
                continue;
            }
            else{
                stack.push(s.charAt(i));
            }
        }
        StringBuilder ans= new StringBuilder(stack.size());
        Iterator value = stack.iterator();
        while(value.hasNext()) {
            ans.append(value.next());
        }
        return ans.toString();
        
    }
}

File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.py
class Solution:
    def removeStars(self, s: str) -> str:
        stack=[]
        for i in range(len(s)):
            if len(stack)>0 and s[i]=='*':
                stack.pop()
                continue
            else:
                stack.append(s[i])
        ans=""
        for i in range(len(stack)):
            ans+=stack[i]
        return ans
            
                
                
        